common/darwin/Classes/FlutterWebRTCPlugin.m

```diff
@@ -91,9 +91,8 @@ - (instancetype)initWithChannel:(FlutterMethodChannel *)channel
 #if TARGET_OS_IPHONE
     _preferredInput = AVAudioSessionPortHeadphones;
     _speakerOn = NO;
-    [AudioUtils setSpeakerphoneOn:_speakerOn];
-    AVAudioSession *session = [AVAudioSession sharedInstance];
-    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(didSessionRouteChange:) name:AVAudioSessionRouteChangeNotification object:session];
+   AVAudioSession *session = [AVAudioSession sharedInstance];
+   [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(didSessionRouteChange:) name:AVAudioSessionRouteChangeNotification object:session];
```

Comment:
Maybe it will be better to use `RTCAudioSession` [`addDelegate`](https://github.com/webrtc-sdk/webrtc/blob/9dab54fd76cb5d0a1a10fe6168cb356f392c502f/sdk/objc/components/audio/RTCAudioSession.h#L207)/[`removeDelegate`](https://github.com/webrtc-sdk/webrtc/blob/9dab54fd76cb5d0a1a10fe6168cb356f392c502f/sdk/objc/components/audio/RTCAudioSession.h#L209) instead of direct `AVAudioSession` usage? But again, it seems that this modification is for another MR?